google/cloud/spanner/keys.h

```diff
@@ -258,6 +264,13 @@ class KeySet {
     }
   }
 
+  /// @name Equality
+  /// Order is of keys and key ranges in the `KeySet` is considered.
```

Comment:
The wording on the "ordering..." comment is a little funny. Maybe remove "is"?

Reply:
Done.